src/modules/household/components/elements/AddToHouseholdButton.tsx

```diff
@@ -62,7 +62,7 @@ const AddToHouseholdButton = ({
   const { openFormDialog, renderFormDialog } =
     useFormDialog<SubmittedEnrollmentResultFieldsFragment>(memoedArgs);
 
-  const { clientAlerts } = useClientAlerts({ client }, false);
+  const { clientAlerts } = useClientAlerts({ client }, false, false);
```

Comment:
This is OK but for improved readability I would change it to accept an object:
`useClientAlerts({ client, householdId,  showClientName, showDeleteButton })`
so this becomes `useClientAlerts({ client, showClientName: false, showDeleteButton: false })` (unless false are the default?)

Reply:
False is the default for both, but I agree with you that it's clearer if these are in an object.
There's already an interface defined `ClientAlertParams` that I just added the two booleans onto, let me know if you like that approach? I also removed them when they are false -- so the default is just used -- although, I don't know if this technicality matters, but they will actually be `undefined` instead of `false` if not provided because I don't think we can give default values on a typescript interface.

Reply:
Yep. That should be fine